### PR TITLE
ETL-475: Check box update

### PR DESCRIPTION
**Pull request by TeaEntry**
https://jira.enigmatry.com/browse/ETL-475

### CODE REVIEW

File: projects/entry-table/src/styles/modules/components/tables/entry-table-nested/rows/_general.scss [hunk 1]
```diff
@@ -12,14 +12,6 @@
             .mat-cell, a {
                 color: map.value-from($theme, 'row.selected-color');
             }
-
```
Comment (typhoon41) [inline projects/entry-table/src/styles/modules/components/tables/entry-table-nested/rows/_general.scss:15]: If project is using tables only and not forms, how will checkboxes in table look like?

File: projects/entry-table/src/styles/modules/components/tables/entry-table-nested/rows/_general.scss [hunk 2]
```diff
@@ -12,14 +12,6 @@
             .mat-cell, a {
                 color: map.value-from($theme, 'row.selected-color');
             }
-    
-            .mat-checkbox-checked .mat-checkbox-background {
-                background-color: map.value-from($theme, 'checkbox.background-color-selected');
```
Comment (typhoon41) [inline projects/entry-table/src/styles/modules/components/tables/entry-table-nested/rows/_general.scss:17]: So we had "background-color-selected" property, we just had to set it in EBO?

File: projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss [hunk 3]
```diff
@@ -21,10 +21,26 @@
...
+        &.mat-checkbox-checked .mat-checkbox-background .mat-checkbox-checkmark-path {}
+
         &:hover {
             .mat-checkbox-frame {
                 border: map.value-from($theme, 'checkbox.border-hover');
             }
         }
     }
+
+    &.mat-checkbox-checked {
```
Comment (typhoon41) [inline projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss:33]: Not sure why form needs this all of this suddenly?

File: projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss [hunk 4]
```diff
@@ -21,10 +21,26 @@
...
+		.mat-checkbox-background {
+			background-color: map.value-from($theme, 'checkbox.selected-background-color');
+
+			.mat-checkbox-checkmark-path {
+				stroke: map.value-from($theme, 'checkbox.selected-checkmark-color')!important;
```
Comment (typhoon41) [inline projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss:38]: What's happening with your stylelint? Formatting seems inconsistent.

**Review by typhoon41 — CHANGES_REQUESTED** [review]

File: projects/entry-table/src/styles/modules/components/tables/entry-table-nested/rows/_general.scss (hunk 2 above)
Comment (TeaEntry) [reply projects/entry-table/src/styles/modules/components/tables/entry-table-nested/rows/_general.scss:17]: yes

File: projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss (hunk 3 above)
Comment (TeaEntry) [reply projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss:33]: if we want to have selected state then yes

File: projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss (hunk 4 above)
Comment (TeaEntry) [reply projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss:38]: don't know :/

File: projects/entry-table/src/styles/modules/components/tables/entry-table-nested/rows/_general.scss (hunk 1 above)
Comment (TeaEntry) [reply projects/entry-table/src/styles/modules/components/tables/entry-table-nested/rows/_general.scss:15]: inital i guess. then we would need to make custom styles in the project. other, not-good-at-all option is to duplicate styles for checkbox here as well. unless you have some other solution

File: projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss (hunk 4 above)
Comment (TeaEntry) [reply projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss:38]: created tickets for that

File: projects/entry-table/src/styles/modules/components/tables/entry-table-nested/rows/_general.scss (hunk 1 above)
Comment (typhoon41) [reply projects/entry-table/src/styles/modules/components/tables/entry-table-nested/rows/_general.scss:15]: For time being, we need to duplicate styles to ensure backwards compatibility.
For future, we need to gather all duplication and brainstorm what to do with it.

File: projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss [hunk 5]
```diff
@@ -21,10 +21,26 @@
             stroke: map.value-from($theme, 'checkbox.checkmark-color')!important;
         }
 
+        &.mat-checkbox-checked .mat-checkbox-background .mat-checkbox-checkmark-path {}
```
Comment (typhoon41) [inline projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss:24]: Empty selector?

**Review by typhoon41 — APPROVED** [review]

**Pull request merged**